tests/client/amo/components/TestOverallRating.js

```diff
@@ -132,22 +141,33 @@ describe('OverallRating', () => {
         });
 
         mockApi
-          .expects('postRating')
+          .expects('submitReview')
           .withArgs(params)
           .returns(Promise.resolve(ratingResponse));
 
-        return actions.createRating({ ...params, userId }).then(() => {
-          assert.equal(dispatch.called, true);
-          const action = dispatch.firstCall.args[0];
-
-          assert.equal(action.type, SET_REVIEW);
-          assert.equal(action.data.addonId, params.addonId);
-          assert.equal(action.data.userId, userId);
-          assert.deepEqual(action.data.rating, ratingResponse.rating);
-          assert.deepEqual(action.data.versionId, ratingResponse.version.id);
-
-          mockApi.verify();
-        });
+        return actions.createRating(
+          {
```

Comment:
So I didn't think the 4 spaces bothered me and then I tried to come up with examples to avoid it and now it bothers me.

I knew we had used the curly on the `return` line before so I went grepping through the code to find how we did it in the past and we only indent by two:

```js
return actions.createRating({
  foo,
});
```

This seems right to me, so following our convention of indenting our `.then()`s by two you'd get:

```js
return actions.createRating({
  foo,
})
  .then(() => ...);
```

Which I agree looks a little odd at first but I think it makes sense (yes there is at least one example of this in the code, I'm not sure if there are more or how intentional it was).

Another nice thing I noticed is that this keeps our objects the same on returns whether they're wrapped in a function or not.

```js
return {
  foo,
};
return bar({
  foo,
});
// Definitely not this, it's a SyntaxError
return
  {
    foo,
  };